packages/thriver-masthead/lib/templates/masthead.html

```diff
@@ -2,7 +2,7 @@
     {{#if isAdmin}}
         {{> sectionAdmin}}
     {{/if}}
-    <section class="masthead" id="{{anchor name=name}}" data-id="{{_id}}">
+    <section class="masthead" id="masthead" data-id="{{_id}}">
```

Comment:
Well it was returning a blank id.
That is the goal, however: I'll use the selector provided.